easybuild/easyconfigs/c/cdat_info/cdat_info-8.2.1-GCCcore-9.3.0-Python-3.8.2.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'PythonPackage'
+
+name = 'cdat_info'
```

Comment:
Is it worth installing this separately, or should it be included with CDAT itself?

Reply:
That's possible only if I fold `unidata` into CDAT as well, but I might as well.